recipes/serial/all/test_package/conanfile.py

```diff
@@ -0,0 +1,16 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package"
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def test(self):
+        if not tools.cross_building(self):
+            bin_path = os.path.join("bin", "test_package")
+            self.run("%s -s" % bin_path, run_environment=True)
```

Comment:
Command line is not read from

```suggestion
            self.run(bin_path, run_environment=True)
```

Reply:
Thx, copy-paste stuff :)